third_party/CMakeLists.txt

```diff
@@ -8,19 +8,38 @@ endif()
 # unity builds for this folder, even if the user manually activated it.
 set(CMAKE_UNITY_BUILD "OFF")
 
-# Build sql-parser
+# Add Libraries with their own build setups
```

Comment:
Should we maybe add one, two sentences of highlevel documentation that there are libraries with built-in CMake support and others that we build manually that will be defined below?

Reply:
done